src/BugsnagUnity/Native/Fallback/NativeClient.cs

```diff
@@ -42,7 +42,7 @@ private void AddIsLaunching(AppWithState app)
             bool isLaunching;
             if (Configuration.LaunchDurationMillis == 0)
             {
-                isLaunching = _launchMarkedAsCompleted;
+                isLaunching = !_launchMarkedAsCompleted;
```

Comment:
This one feels like something an automated test should have found?

Reply:
added tests for both setting 0 and setting 0 and calling MarkLaunchComplete 👍